Mathlib/Algebra/Algebra/Subalgebra/Unitization.lean

```diff
@@ -208,10 +208,36 @@ variable {R : Type*} [NonAssocSemiring R]
 def Subsemiring.toNonUnitalSubsemiring (S : Subsemiring R) : NonUnitalSubsemiring R :=
   { S with }
 
+theorem Subsemiring.toNonUnitalSubsemiring_injective :
+    Function.Injective (toNonUnitalSubsemiring : Subsemiring R → _) :=
+  fun S₁ S₂ h => SetLike.ext'_iff.2 (
+    show (S₁.toNonUnitalSubsemiring : Set R) = S₂ from SetLike.ext'_iff.1 h)
+
+@[simp]
+theorem Subsemiring.toNonUnitalSubsemiring_eq {S₁ S₂ : Subsemiring R} :
```

Comment:
```suggestion
theorem Subsemiring.toNonUnitalSubsemiring_inj {S₁ S₂ : Subsemiring R} :
```

Reply:
named by analogy to e.g. `Subgroup.toSubmonoid_eq`

Reply:
ah, many precedents exist for both conventions
is there consensus on which is preferred?

Reply:
I have checked and _inj is overwhelmingly preferred. Will make another PR fixing the few _eq names.